modules/core/src/main/java/com/smartbear/readyapi4j/result/TestStepResult.java

```diff
@@ -44,9 +44,10 @@
     List<String> getMessages();
 
     /**
-     * @return the HAR response for the underlying HTTP transaction if this was an HTTP-based TestStep - null otherwise
+     * @return the underlying HAR log entry for this TestStep result if this was an HTTP-based
+     * TestStep, null if no transaction log is available
```

Comment:
Does the Javadoc work if you have a line break in the middle of the return line?